skyportal/facility_apis/atlas.py

```diff
@@ -246,6 +246,9 @@ def get(request, session):
 
         r = requests.post(facility_microservice_url, json=request_body)
         log(f'Response for request {request.id}: {r.text}')
+        data_out = r.json()
+        if 'message' in data_out:
+            request.status = data_out['message']
```

Comment:
This seems wrong, usually the status is supposed to be 200 or 400, etc.

Reply:
For whatever reason, we have used status as the text.

Reply:
What is this request object that is being passed in? It would be strange to assign to the request if it were a web request so I presume it must be something else.